client/lib/performance-tracking/components/performance-tracker-stop.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { isSingleUserSite, isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import isSiteStore from 'state/selectors/is-site-store';
+import { stopNavigation } from '../api';
+
+class PerformanceTrackerStop extends React.Component {
```

Comment:
Any reason not to make this a function style component?

Reply:
Not really, I can do a function component with `useEffect`, it should be the same behaviour